doc/man7/OSSL_PROVIDER-FIPS.pod

```diff
@@ -189,6 +204,21 @@ TEST-RAND is an unapproved algorithm.
 
 =back
 
+=head1 Provider Parameters
+
+See L<provider-base(7)/Provider parameters> for a list of base parameters.
+Additionally the OpenSSL FIPS provider also supports the following gettable
+parameters:
+
+=over 4
+
+=item "security-checks" (B<OSSL_OSSL_PROV_PARAM_SECURITY_CHECKS>) <unsigned integer>
+
+For further information refer to the L<openssl-fipsinstall(1)> option
+B<-no_security_checks>.
+
+=back
+
```

Comment:
Why this duplication?

Reply:
Merge error.... Fixed.
Added a CHANGES entry also.